rpc/shared/shared/src/test/scala/io/udash/rpc/UPickleUdashRPCFramework.scala

```diff
@@ -135,7 +135,7 @@ object UPickleUdashRPCFramework extends UdashRPCFramework {
     }
   }
 
-  implicit val RawValueCodec: GenCodec[RawValue] = new GenCodec[Js.Value] {
+  implicit val RawValueCodec: GenCodec.Auto[RawValue] = GenCodec.Auto(new GenCodec[Js.Value] {
```

Comment:
Same here - ideally you shouldn't ever need to declare any implicit of type `GenCodec.Auto`. Instead, declare a `GenCodec` and if you ask for `GenCodec.Auto`, it will wrap the existing `GenCodec`.

I mean, this should work that way - if it doesn't, let me know.